config.md

```diff
@@ -905,6 +905,18 @@ Symbol prawego nawiasu w oknie stanów, może być ciąg znaków
 
 ---
 
+## `ateam.options.leader_indicator`
```

Comment:
Proponuję usunąć to ustawienie. Drugie niech będzie puste jako default.

Reply:
makes sense, zbouncuj i poprawke zaraz wrzuce

Reply:
Nie trzeba, po prostu zaktualizuj branch.

Reply:
done